wire-ios/Wire-iOS/Sources/UserInterface/Location/MapViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import MapKit
+
+// MARK: - MapManagerDelegate
+
+protocol MapViewControllerDelegate: AnyObject {
+
+    func mapViewController(_ viewController: MapViewController, didUpdateUserLocation userLocation: MKUserLocation)
+    func mapViewController(_ viewController: MapViewController, regionDidChangeAnimated animated: Bool)
+    func mapViewControllerDidFinishRenderingMap(_ viewController: MapViewController, fullyRendered: Bool)
+
+}
+
+// MARK: - MapViewController
+
+final class MapViewController: NSObject {
+
+    // MARK: - Properties
+
+    let mapView = MKMapView()
```

Comment:
I think it's weird that we have this view controller, which is actually never presented and doesn't use its `view`, and also that it owns another view (`mapView`) which is presented somewhere else.

Reply:
Sorry, I got confused by the name MapViewController. I thought it's a subclass of UIViewController.